packages/elements-dev-portal/src/hooks/useGetNodes.ts

```diff
@@ -10,22 +10,24 @@ export function useGetNodes({
   search,
   workspaceId,
   projectIds,
+  branchSlug,
```

Comment:
`branch`?

Reply:
yeah, that works I think. I actually started with `branch` but then went with `branchSlug` as this is how the query param was called